custom_reports/http_api/definitions/query_request.yaml

```
@@ -28,14 +28,22 @@ properties:
     example: [{"column_id":"device_type"},{"column_id":"revenue","transformation_id":"sum"}]
   date_from:
     type: string
-    format: date
-    description: Start date for the query (inclusive). Cannot be used with relative_date field at
-      the same time. Mandatory if ``relative_date`` is not used.
+    oneOf:
+      - format: date
+        description: Start date for the query (inclusive).
```

```suggestion
        description: Start date for the query (inclusive, in website's timezone).
```
dates are also in the timezone, and clients often ask about it